tests/contrib/anthropic/test_anthropic_llmobs.py

```diff
@@ -30,6 +32,29 @@
     "ddtrace_global_config", [dict(_llmobs_enabled=True, _llmobs_sample_rate=1.0, _llmobs_ml_app="<ml-app-name>")]
 )
 class TestLLMObsAnthropic:
+    def test_completion_proxy(self, anthropic, ddtrace_global_config, mock_llmobs_writer, mock_tracer, request_vcr):
```

Comment:
Just because I dislike over-nested code, can we use `@mock.patch("anthropic.SyncAPIClient.post")`  as a decorator argument instead?